cmd/kops/create.go

```diff
@@ -76,7 +76,7 @@ func NewCmdCreate(f *util.Factory, out io.Writer) *cobra.Command {
 		Example: createExample,
 		Args:    cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return RunCreate(context.TODO(), f, out, options)
+			return RunCreate(cmd.Context(), f, out, options)
```

Comment:
Doesn't this return `nil` as we don't run with `cobraCommand.ExecuteContext`?

Reply:
Ugh I hope it returns context.TODO() or context.Background() but I'll check - thanks for pointing it out!

Reply:
I think it actually defaults to context.Background, but I switched us to call ExecuteContext so it's clearer (which might also allow us to reply to control-C more promptly in future :-) )